content/projects/web-development-website/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+content_type: project
+flavours:
+- none
+learning_outcomes:
+- web_dev_hosting
+- web_dev_responsive_page_design
+- web_dev_web_page_structure
+- web_dev_web_design_principals
+prerequisites:
+  hard:
+  - topics/free-code-camp/responsive-web-design/1-basic-html-and-html-5
+  - topics/free-code-camp/responsive-web-design/2-basic-css
+  - topics/what-to-put-in-your-cv
+  - topics/web-frontend/intro-to-web-design/
```

Comment:
Can we add the simple website as a prerequisite and make them just update that? Otherwise the web devs will end up doing 2 personal portfolio projects